src/api/twitterApi.ts

```diff
@@ -0,0 +1,98 @@
+// see https://developer.twitter.com/en/docs/twitter-api/users/lookup/api-reference/get-users-by-username-username#tab1
+// https://github.com/twitterdev/twitter-api-typescript-sdk (library)
+// npm install twitter-api-sdk
+import { Client } from 'twitter-api-sdk'
+
+interface TwitterUser {
+  id: string
+  name: string
+  followersCount: number
+}
+
+interface TwitterPost {
+  id: string
+  retweetCount: number
+  likeCount: number
+}
+
+const twitterClient = new Client('auth-token')
```

Comment:
Please use an environment variable for the auth token (install [dotenv](https://github.com/motdotla/dotenv) and follow the instructions in their docs). Also provide the token you used for development to us via Slack so we can test your implementation.